integration/config/core.yaml

```diff
@@ -539,12 +539,28 @@ chaincode:
     # List of directories to treat as external builders and launchers for
     # chaincode. The external builder detection processing will iterate over the
     # builders in the order specified below.
-    externalBuilders: []
+    # externalBuilders: []
         # - path: /path/to/directory
         #   name: descriptive-builder-name
         #   environmentWhitelist:
         #      - ENVVAR_NAME_TO_PROPAGATE_FROM_PEER
         #      - GOPROXY
+    externalBuilders:
+        # FPC Addition 0: external builder for fpc-c chaincode
+        - path: ../../fabric/externalBuilder
+          name: fpc-c
+          environmentWhitelist:
+              - FPC_HOSTING_MODE
```

Comment:
This is nice!
Not sure about the host mode, I think it should always be docker.
But a similar variable can be used to distinguish between FPC cc and FPC architectural cc (i.e. ERCC).

Reply:
I like the docker-less approach! I suggest we add dockerized external launcher to parking and see if there is need for it from Fabric community.

Reply:
well, from a both deployment and security perspective docker is better.  But it's certainly not on the critical path ...